test/apps/widgets-9.2/package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "deckgl-example-pure-js-basic",
+  "name": "deckgl-example-pure-js-widgets",
```

```suggestion
  "name": "deckgl-example-widgets",
```

Also, I would remove "9.2" from the test app names in general

---

I made sure to test the infographic test app and include new widgets there that now work better than before.

The website examples (deck.gl/examples) tend to be something _interesting_, something that goes beyond a functional example. I'd love to make a dedicated example.. but haven't yet.. Weather ballon and chase vehicle data could be cool.

We could also try integrating widgets into existing website examples where it makes sense.

Finally, _functional_ embedded examples in the api-reference doc pages (like we have for layers) would be great too. The widget previewer is a step towards this, but we could do more.